### PR TITLE
freeradius3: update where to download freeradius-3.0.26

### PR DESCRIPTION
**Pull request by predators46**
Maintainer:
Compile tested: armsr
Run tested: armv8

Description:

update where to download freeradius3

add modules

freeradius3-mod-totp
freeradius3-mod-sql-unixodbc
freeradius3-mod-eap-sim
freeradius3-mod-cache
freeradius3-mod-cache-rbtree

### CODE REVIEW

File: net/freeradius3/Makefile [hunk 1]
```diff
@@ -8,19 +8,19 @@
...
-PKG_RELEASE:=4
+PKG_VERSION:=3.0.26
+PKG_RELEASE:=5
 
-PKG_SOURCE:=release_$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/FreeRADIUS/freeradius-server/tar.gz/release_$(PKG_VERSION)?
-PKG_HASH:=6aea98d6126035e7ccca483d8b3faea447030169639807017ec98985b78fb2ca
+PKG_SOURCE:=freeradius-server-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=ftp://ftp.freeradius.org/pub/freeradius/
+PKG_HASH:=9a65314c462da4d4c4204df72c45f210de671f89317299b01f78549ac4503f59
```
Comment (BKPepe) [inline net/freeradius3/Makefile:16]: This is not explained why. You just simply stated "update where to donwload freeradius". Thats not enough.

File: net/freeradius3/Makefile [hunk 2]
```diff
@@ -136,6 +136,23 @@ define Package/freeradius3-mod-attr-filter/conffiles
...
+define Package/freeradius3-mod-cache/conffiles
+/etc/freeradius3/mods-available/cache
+/etc/freeradius3/mods-available/cache_auth
+endef
+
+define Package/freeradius3-mod-cache-rbtree
+  $(call Package/freeradius3/Default)
+  DEPENDS:=freeradius3 +freeradius3-mod-cache
+  TITLE:=Cache Drivers rbtree
+endef
```
Comment (BKPepe) [inline net/freeradius3/Makefile:154]: Why? This is completely unrelated to your commit "update where to download XY"

Comment (BKPepe) [reply net/freeradius3/Makefile:154]: Applies to all your new packages. That should be done in separate commit IMHO

**Review by BKPepe — CHANGES_REQUESTED** [review]

File: net/freeradius3/Makefile [hunk 3]
```diff
@@ -8,19 +8,19 @@
...
-PKG_VERSION:=3_0_26
-PKG_RELEASE:=4
+PKG_VERSION:=3.0.26
+PKG_RELEASE:=5
 
-PKG_SOURCE:=release_$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/FreeRADIUS/freeradius-server/tar.gz/release_$(PKG_VERSION)?
-PKG_HASH:=6aea98d6126035e7ccca483d8b3faea447030169639807017ec98985b78fb2ca
+PKG_SOURCE:=freeradius-server-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=ftp://ftp.freeradius.org/pub/freeradius/
```
Comment (BKPepe) [inline net/freeradius3/Makefile:15]: Why did you decided to use FTP instead of HTTPS?
There are better possibilites. Stay with GitHub or rather use https://freeradius.org/ftp/pub/freeradius/

**Review by BKPepe — CHANGES_REQUESTED** [review]

**Comment by predators46** [comment]
I'll close this and update as you wish

**Comment by BKPepe** [comment]
Dont create a new PR. Use the existing one.

Dne ne 10. 12. 2023 13:38 uživatel Predators ***@***.***>
napsal:

**Comment by predators46** [comment]
@BKPepe 
maybe next year I'll update this
